lib/flagsmith/version.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+module Flagsmith
+  VERSION = '3.0.0'
```

Comment:
Is there a way to define this only once? I can see that we have defined it here as well as in the `Gemfile.lock`, although perhaps that is autogenerated?

Reply:
This is just one place where the version is defined. `Gemfile.lock` is an auto-generated file.